Tests/Test_Renode_MotorSimulator.c

```diff
@@ -0,0 +1,136 @@
+#include "os.h"
+#include "CANbus.h"
+#include <stdio.h>
+#include "Tasks.h"
+
+// Global variables
+static float velocity; 
+static float current; 
+static float total_accel;
+static CANDATA_t oldCD;
+static int ctr = 0;
```

Comment:
Make sure all ints are used with the fixed width integer types (uint8_t, uint16_t, etc.) so that we don't reserve more space than we need.